proto/ligato/generic/options.proto

```diff
@@ -1,4 +1,4 @@
-syntax = "proto2";
+syntax = "proto3";
```

Comment:
> CI proto check is failing (https://github.com/ligato/vpp-agent/actions/runs/4828274020/jobs/8601897603?pr=1925)

This is probably the culprit, because it is a breaking change. I think it was overlooked when migrating from proto v2 to proto v3, so I changed it now. Should not really affect anything, but I can revert this back to proto2 if needed.

Reply:
Ok, so the proto lint check is broken. I guess we can either fix it in this PR (if you see some easy solution) or what imho would be cleaner (in case of complicated/nonobvious solution) to revert back to proto2(and have lint capability) and make a issue that handles this exact problem in another PR.

Reply:
Hmmm, I don't think the lint check is broken, it seems to be supposed to fail when it finds a breaking change in `.proto` files. So it found one and it failed. Probably to make sure breaking proto changes are not merged without discussion (maybe not merged at all)? @ondrej-fabry any thoughts on this?

Reply:
@pemoticak You are right. I misread the log (https://github.com/ligato/vpp-agent/actions/runs/5012550482/jobs/8984628112?pr=1925). The `buf check lint` didn't fail. The `buf check breaking` running after that failed. So it is only check for making breaking changes in proto files. However, it is ok to change proto2 to proto3 in this case so we can ignore that check fail.